pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -146,7 +146,8 @@ let
         done
 
         # Remove tests because they add a runtime dependency on gcc
-        rm -rf $out/share/zfs/zfs-tests
+        # TODO(mindavi): Should we optionally remove these? How does that work?
+        #rm -rf $out/share/zfs/zfs-tests
```

Comment:
Just put them in a different output.

Reply:
Couldn't get this to work properly, so a different method is used now. Have 1 last idea to get it working, but let's see if that pans out.